net-libs/gnutls/metadata.xml

```diff
@@ -16,7 +16,7 @@
 			Add support for PKCS#11 through <pkg>app-crypt/p11-kit</pkg>
 		</flag>
 		<flag name="tools">
-			Build extra tools
+			Build cli tools such as gnutls-cli, certtool and oscptool
```

Comment:
While I like this change, it feels totally random on this PR. You can leave it, it just seems like two separate PRs got mixed into one with your workflow.